acceptance/openstack/networking/v2/extensions/vpnaas/ikepolicy_test.go

```diff
@@ -0,0 +1,24 @@
+// +build acceptance networking vpnaas
+
+package vpnaas
+
+import (
+	"testing"
+
+	"github.com/gophercloud/gophercloud/acceptance/clients"
+	"github.com/gophercloud/gophercloud/acceptance/tools"
+)
+
+func TestPolicyCRUD(t *testing.T) {
```

Comment:
As mentioned in the IPSec PR, this test name clashes so we'll want to change it.